partials/_debug-grid.css.js

```diff
@@ -14,4 +14,6 @@ module.exports = `
 .debug-grid-16-solid {
   background:white url(data:image/gif;base64,R0lGODdhEAAQAPEAAADw/wDx/xXy/////ywAAAAAEAAQAAACIZyPKckYDQFsb6ZqD85jZ2+BkwiRFKehhqQCQgDHcgwEBQA7) repeat top left;
 }
+
+.bg-primary-gradient { background: linear-gradient(90deg, #00A884 0%, #00A830 77.67%); }
```

Comment:
Put this in `_styles.css.js` plss, seems to be the _custom things_ place to me

Reply:
That was my idea, but it didn't work. I didn't wanna waste time debugging so I found the closest background place that works. If you think this is to dirty I can spend some more time on it.